README.md

```diff
@@ -23,25 +23,26 @@
 #### **JSON formatted example**
 ```
 {
-    {
-        "City": cityName,
-        "StartDate": startDate,
-        "EndDate": endDate, 
-        "AverageHighTemperature": averageHighTemperature,
-        "AverageLowTemperature": averageLowTemperature,
+    "response": {
+        "city": cityName,
+        "startDate": startDate("yyyy/mm/dd"),
```

Comment:
I am sorry, but I did not notice it from the first review but the format is not consistent with the one from the sample response
It should be : yyyy-mm-dd

Reply:
I will just merge it in and quickly fix the format later